.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Docs to PDF
+# This workflow is triggered on pushes to the master branch.
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  converttopdf:
+    name: Create our PDF
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Generate PDF for each folder in /contents
+      - uses: mkrakowitzer/actions-makepdfs@master
+        with:
+          markdown_dir: 
+          output_dir: pdfen
```

Comment:
I'm not sure I understand the name of the folder...

Reply:
@sapegin 

I recommend that you read through the documentation at https://github.com/marketplace/actions/create-pdf

Reply:
I don't this it will explain the choice of names in this pull request 🤔